examples/OneOpenAir/OneOpenAir.ino

```diff
@@ -1051,7 +1051,7 @@ static void updatePm(void) {
       Serial.printf("[1] Relative Humidity: %d\r\n", measurements.hum_1);
       Serial.printf("[1] Temperature compensated in C: %0.2f\r\n",
                     ag->pms5003t_1.temperatureCompensated(measurements.temp_1));
-      Serial.printf("[1] Relative Humidity compensated: %f\r\n",
+      Serial.printf("[1] Relative Humidity compensated: %0.2f\r\n",
```

Comment:
The methods below should be renamed to compensateTemp / compensateHum and also documented. I mean we need to explain what the formula is, maybe link to some external document


float PMS5003TBase::temperatureCompensated(float temp) {
  if (temp < 10.0f) {
    return temp * 1.327f - 6.738f;
  }
  return temp * 1.181f - 5.113f;
}

float PMS5003TBase::humidityCompensated(float hum) {
  hum = hum * 1.259f + 7.34f;

  if (hum > 100.0f) {
    hum = 100.0f;
  }
  return hum;
}




float PMS5003TBase::temperatureCompensated(float temp) {
  if (temp < 10.0f) {
    return temp * 1.327f - 6.738f;
  }
  return temp * 1.181f - 5.113f;
}

float PMS5003TBase::humidityCompensated(float hum) {
  hum = hum * 1.259f + 7.34f;

  if (hum > 100.0f) {
    hum = 100.0f;
  }
  return hum;
}

Reply:
> The methods below should be renamed to compensateTemp / compensateHum and also documented. I mean we need to explain what the formula is, maybe link to some external document

@nick-4711 OK

@airgradienthq  Can you provide the link document for the formula.

Reply:
We can link to 

https://www.airgradient.com/documentation/correction-algorithms/